src/core/operations/Sort.mjs

```diff
@@ -131,6 +133,48 @@ class Sort extends Operation {
         return a.localeCompare(b);
     }
 
+    /**
+     * Comparison operation for sorting of hexadecimal values.
+     *
+     * @author Chris van Marle
+     * @private
+     * @param {string} a
+     * @param {string} b
+     * @returns {number}
+     */
+    static _hexadecimalSort(a, b) {
+        const a_ = a.split(/([^\da-f]+)/i),
+            b_ = b.split(/([^\da-f]+)/i);
+
+        for (let i = 0; i < a_.length; ++i) {
+            const t = parseInt(a_[i], 16);
+            if (!isNaN(t)) {
```

Comment:
could using a `typeof` test be more clear than the double-negative `!isNaN`? e.g. `if(typeof t === "string")`

Reply:
parseInt returns `NaN` when it fails to parse the int. The only way to check that afaik is with `isNaN`.